packages/smart-wallet/test/supports.js

```diff
@@ -88,17 +88,11 @@ const makeFakeBridgeManager = () =>
         },
         toBridge(obj) {
           assert(handler, `No handler for ${bridgeId}`);
-          switch (obj.type) {
-            case ActionType.WALLET_ACTION:
-            case ActionType.WALLET_SPEND_ACTION: {
-              // @ts-expect-error handler possibly undefined
-              return E(handler).fromBridge(obj);
-            }
-
-            default: {
-              assert.fail(`Unsupported bridge object type ${obj.type}`);
-            }
-          }
+          // Pass without validation. Production use will only pass through
+          // valid types, but the smart wallet contract should also validate
+          // in the even an invalid one comes through.
```

Comment:
I'm struggling to understand this comment. How is "Production" relevant?

My understanding is...

```suggestion
          // Rely on interface guard for validation as usual.
```

Reply:
I agree "production" is confusing. I meant "the bridge manager that is not fake" (since this is makeFakeBridgeManager).

Do you think it's better not to mention that the real bridge manager does its own validation? I agree this shouldn't trust that to happen but I thought it worth pointing out that it does. How about this…

> Rely on interface guard for validation.
> This should also be validated upstream but don't rely on it.

Reply:
Oh. I completely missed the test context!

Reply:
"Pass thru without validation for testing"?